.github/workflows/e2e.yml

```diff
@@ -0,0 +1,71 @@
+---
+name: E2E-CI
+
+on:  # yamllint disable-line rule:truthy
+  push:
```

Comment:
If you want this job to run as a CI of each PR you should also add `pull_request` here.